src/CoreBundle/private/library/classes/TCMSLogChange.class.php

```diff
@@ -1597,9 +1598,10 @@ public static function deleteVirtualNonDbExtension($iLine, $sEntryPoint, $sClass
             throw new ErrorException("unable to find virtual extension {$sClassName} for {$sEntryPoint} (called from the running update in line {$iLine})", 0, E_USER_ERROR, __FILE__, __LINE__);
         }
 
-        $tableEditorManager = TTools::GetTableEditorManager($oExtension->table, $oExtension->id);
-        $tableEditorManager->AllowDeleteByAll(true);
-        $tableEditorManager->Delete($oExtension->id);
+        $connection = self::getDatabaseConnection();
+
+        $query = 'DELETE FROM '.$connection->quoteIdentifier($oExtension->table).' WHERE id = '.$connection->quote($oExtension->id);
```

Comment:
Could we work with a parameter for the ID? Although we need string concatenation for the table name, we should use this as little as possible.